jkube-kit/enricher/api/src/main/java/org/eclipse/jkube/kit/enricher/handler/ServiceHandler.java

```diff
@@ -43,7 +43,7 @@ public List<Service> getServices(List<ServiceConfig> services) {
             ServiceBuilder serviceBuilder = new ServiceBuilder()
                 .withNewMetadata()
                   .withName(service.getName())
-                  .withAnnotations(getAnnotations(service))
+                  .withAnnotations(new HashMap<>())
```

Comment:
I think we don't need this either, [ObjectMeta](https://github.com/fabric8io/kubernetes-client/blob/main/kubernetes-model-generator/kubernetes-model-core/src/generated/java/io/fabric8/kubernetes/api/model/ObjectMeta.java#L59) seems to instantiate annotations with empty map already.
```suggestion
```